README.md

```diff
@@ -10,6 +10,18 @@ Opinionated stacks of ready-to-run Jupyter applications in Docker.
 If you're familiar with Docker, have it configured, and know exactly what you'd like to run, this one-liner should work in most cases:
```

Comment:
Probably need to update this lead-in sentence since now there's a few one-liners. (And thanks for that!)